SConstruct

```diff
@@ -1582,17 +1582,24 @@ end program main
 
 set_fortran("{}FLAGS", env["FORTRANFLAGS"])
 
-if env["using_apple_clang"] and env["f90_interface"] == "default":
+if (env["using_apple_clang"] and env["f90_interface"] == "default" and
+    not env["FORTRAN"]):
     env["f90_interface"] = "n"
 
 if env['f90_interface'] in ('y','default'):
     foundF90 = False
-    if env['FORTRAN']:
+    fortran_used = ""
+    if env['FORTRAN'] == "mpifort":
```

Comment:
Should this be `if "mpifort" in env["FORTRAN"]` to match the other cases? Like what if the absolute path to `mpifort` is passed

Reply:
Good catch!